src/EnergyPlus/FluidProperties.cc

```diff
@@ -4744,7 +4744,12 @@ namespace FluidProperties {
                     break; // LoopC DO loop
                 }
             }
-            if (std::abs(RawConcData(HiIndex) - RawConcData(HiIndex - 1)) >= ConcToler) {
+
```

Comment:
Add a condition for single-concentration case, so it could be processed properly. In this scenario, it will just take that set of data. 
This might not be an uncommon scenario for supplying just one concentration of data, or just have fluid of a single concentration to use.

Reply:
I have a strong suspicion that if I look too hard at the code around this change, I'm going to find lots of things I want to change.  So I'll try to keep my review to just these lines.